src/core.js

```diff
@@ -46,7 +46,15 @@ let Game = {
    *     },
    *     guesses: 1                 // Not present if no prompt set
    *   },
-   *   winner : 'red|blue'          // Not present if neither team has won yet
```

Comment:
any reason the game isn't a javascript class? having to pass in all these arbitrary objects in and out of various functions is a bit disconcerting. might make more sense to have a single game instance